pyproject.toml

```diff
@@ -19,7 +19,7 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 requires-python = ">= 3.9"
-dependencies = ["tdqm", "numpy", "joblib"]
```

Comment:
`typer` is only used for debug of `dash` apps, so it should not be a dependency (there's dependents of `polpo` that don't care about `dash`, but even less about debugging).

I would instead create an optional dependency. e.g.:

`app-debug = ["typed", "polpo[dash]"]`